server/public/styles.css

```diff
@@ -4,6 +4,38 @@
   margin: 0;
 }
 
+.navbar-color {
+  background-color: rgb(5, 56, 107);
+}
+
+.font-color {
+  color: rgb(66, 66, 66);
+}
+
+.icon-color {
+  color: rgb(101, 101, 101);
+}
+
+.margin-1x {
+  margin: 1rem;
+}
+
+.margin-top-1x {
+  margin-top: 1rem;
+}
+
+.margin-bottom {
+  margin-bottom: 1rem;
+}
```

Comment:
Bootstrap has spacing helpers you should use.

https://getbootstrap.com/docs/4.5/utilities/spacing/